runtime/ftplugin/qf.vim

```diff
@@ -14,3 +14,12 @@ let b:undo_ftplugin = "set stl<"
 
 " Display the command that produced the list in the quickfix window:
 setlocal stl=%t%{exists('w:quickfix_title')?\ '\ '.w:quickfix_title\ :\ ''}\ %=%-15(%l,%c%V%)\ %P
+
+autocmd Syntax <buffer>
```

Comment:
The original lines set by Vim.  It felt cleaner than using syntax and conceal.  I secretly hate the way the location list renders.